dGame/dBehaviors/BasicAttackBehavior.cpp

```diff
@@ -189,15 +188,15 @@ void BasicAttackBehavior::DoBehaviorCalculation(BehaviorContext* context, RakNet
 		return;
 	}
 
-	const float ImmunityTime = Game::zoneManager->GetWorldConfig()->globalImmunityTime; //DETERMINE THE DAMAGE COOLDOWN TIME
-	LOG_DEBUG("Damage cooldown time is  %f s",destroyableComponent->GetDamageCooldownTimer()); //ALSO PRINT CURRENT TIMER VALUE
+	const float immunityTime = Game::zoneManager->GetWorldConfig()->globalImmunityTime; //Determine the damage cooldown time
+	LOG_DEBUG("Damage cooldown time is %f s", destroyableComponent->GetDamageCooldownTimer()); //Print current damage cooldown time value
```

Comment:
Thinking on it now, this log is going to be pretty noisy since we use it in all 3 paths and its a value that doesn't change.  Do you think we should remove this or keep it in?

Reply:
imo, remove it

Reply:
Just to clarify, this is actually printing the value of the timer whenever damage is taken, not the length of the immunity period (which, as you say, doesn't change). I've found it useful in debugging if the timer is functioning properly and the value is properly being set (or properly being NOT set, in the case of m_DontApplyImmune. Do you still want me to get rid of it?

Reply:
Ah in that case this makes sense to keep for debug logging